The following arguments were used to generate this output.
[-technique, selection, -coverage, function, -order, relative, -origOrder, ambari-new/ambari-server/target/ambari-server-auto-order, -testInputDir, ambari/ambari-server/target/sootTestOutput-auto-selection, -filesToDelete, ambari-new/ambari-server/target/ambari-server-env-files, -project, Ambari-Server, -testType, auto, -oldVersCFG, ambari/ambari-server/target/selectionOutput, -newVersCFG, ambari-new/ambari-server/target/selectionOutput, -getCoverage, -outputDir, /homes/gws/winglam/winglam/ambari/dependent-tests-impact/experiments/selection-results, -timesToRun, 1, -dependentTestFile, /homes/gws/winglam/winglam/ambari/dependent-tests-impact/experiments/selection-dt-list/selection-Crystal-auto-function-relative.txt]

Execution time of TLG and its time to find/nullify any DTs for 1 machine (does not include the time to run the tests in the new order): 0.585
Number of tests selected out of total in original order: 19 / 3337
Number of DTs not fixed: 0
Number of DTs fixed: 0
APFD value: 0.3362406633559423
Execution time for executing the following testing order: 0.374

Test order list:
[RegressionTest2.test020, RegressionTest6.test147, RegressionTest0.test090, RegressionTest0.test331, RegressionTest0.test429, RegressionTest1.test004, RegressionTest2.test176, RegressionTest2.test236, RegressionTest2.test311, RegressionTest2.test338, RegressionTest2.test448, RegressionTest2.test462, RegressionTest4.test302, RegressionTest6.test101, RegressionTest1.test334, RegressionTest2.test388, RegressionTest3.test090, RegressionTest1.test147, RegressionTest3.test068]

Time each test takes to run in the new order:
[187186794, 8511072, 17819620, 10748843, 8876060, 16866617, 17604363, 6450032, 8697430, 9038109, 6000131, 5669724, 12120604, 1348709, 18644361, 15217616, 12470470, 4909916, 5411619]

Dependent test list:
[Test: randoop.crystal.RandoopTest6.test55, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test30], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test488, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test362], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test30, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest6.test5], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest6.test5, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test488], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test410, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test68], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test176, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test167], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test412, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test382], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test180, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test171], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test302, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test206], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test337, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test331], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test359, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test338], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test382, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test359], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test41, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test18], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test324, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test228], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test78, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test45], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test160, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test134], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test93, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test92], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test92, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test77], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test134, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test129], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test180, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test175], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test130, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test80], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test175, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test132], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test18, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test330], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test441, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test414], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test331, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test317], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test411, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test331], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test414, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test411], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test253, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test198], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test50, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test27], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test228, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test221], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test49, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test46], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test447, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test445], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test402, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test255], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test495, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test404], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test319, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test183], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test80, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test78], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test444, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test373], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test255, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test447], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test135, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test85], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test451, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test440], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test317, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test253], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test330, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test319], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test82, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test49], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test206, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test180], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test198, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test43], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test440, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test412], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test45, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test41], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test128, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test420], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test183, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test180], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test43, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test13], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test151, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test135], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test166, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test153], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test171, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test166], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test373, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test132], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test153, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test151], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test77, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test75], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test75, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test48], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test445, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test441], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test104, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test101], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test132, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test128], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test129, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test114], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test382, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test324], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test44, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test26], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test26, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test495], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test46, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test44], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test400, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test382], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test101, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test93], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test12, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test289], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest1.test289, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test410], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test118, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest2.test12], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test221, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test160], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test114, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test109], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test109, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test104], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test420, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test400], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest2.test13, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest1.test444], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest0.test167, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest0.test50], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest4.test85, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest4.test82], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test338, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test337], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test323, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test302], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test132, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test130], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest3.test404, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest3.test402], The revealed different behavior: PASS, when executed after: [], Test: randoop.crystal.RandoopTest5.test331, Intended behavior: FAILURE, when executed after: [randoop.crystal.RandoopTest5.test323], The revealed different behavior: PASS, when executed after: []]

Coverage test list:
[1333, 1333, 666, 666, 666, 666, 666, 666, 666, 666, 666, 666, 666, 0, 0, 0, 0, 0, 0]
--------------------------
Total time (of all machines and iterations plus initial TestListGenerator): 1.835